ease.js

```diff
@@ -65,7 +80,7 @@ function Ease(username, appName, appToken) {
 
   this.subscribe = function(application) {
     if(this.conn == undefined) {
-      this.connect();
+      this.connect(application);
```

Comment:
application can be referred to with `this.appName`. No need to have application passed in as it is redundant.